### PR TITLE
Add example of two-stage bisection

### PR DESCRIPTION
**Pull request by jkeenan**
Based on work in https://github.com/Perl/perl5/issues/19463

### CODE REVIEW

**Comment by jkeenan** [comment]
The p.r. documents what invocations were used in each stage of the bisection, but I feel it could use some improvement as to *why we needed a two-stage process*.

**Review by demerphq — APPROVED** [review]
You could argue this is a *three* stage bisect, as stage 0 would have been Hugo finding the bracketing versions for it, presumably manually, and then bisecting to the find the commit. 

But if you were to argue that stage 0 wasn't true bisection I wouldn't disagree either. :-)

It does make me think that maybe bisect.pl could learn a new trick. :-)

**Comment by jkeenan** [comment]
> You could argue this is a _three_ stage bisect, as stage 0 would have been Hugo finding the bracketing versions for it, presumably manually, and then bisecting to the find the commit.
> 

In the **EXAMPLES** section we already have one instance where we mention that the investigator used pre-built production releases to narrow the scope over which `Porting/bisect.pl` was asked to iterate.  By that precedent *this* instance counts as two stages (two invocations of the program) rather than three.

**Review by hvds — APPROVED** [review]

File: Porting/bisect-runner.pl [hunk 1]
```diff
@@ -927,6 +927,60 @@ =head2 Point in time where code started to segfault is unknown
...
+
+=item * Problem
+
+In C<-DDEBUGGING> builds, the debug flags C<Xvt> would crash a program when F<strict.pm> was loaded via C<require> or C<use>.
+
+=item * Solution
+
+Two-stage solution.  In each stage, to shorten debugging time investigator
+made use of existing set of production releases of F<perl> built with
+C<-DDEBUGGING>.
```
Comment (hvds) [inline Porting/bisect-runner.pl:943]: For the additional explanation you asked about you could add here something like:

> The first bisect determined the point at which code was introduced to F<strict.pm> that triggered the problem. With an understanding of the trigger, the second bisect then determined at what point such a trigger started causing a crash.

**Pull request merged**